test/cookbooks/octopus-deploy-test/recipes/tentacle.rb

```diff
@@ -22,66 +22,47 @@
 tentacle = node['octopus-deploy-test']['tentacle']
 
 # This section will mock out the certificate creation
-directory 'C:\Octopus' do
-  action :create
-end
-
-cookbook_file 'C:\Octopus\tentacle_cert.txt' do
+cert_file = 'C:\tentacle_cert.txt'
```

Comment:
Just shared this file, because of this we don't need to create the home directories before.